blackbox/utils/mixins.py

```diff
@@ -0,0 +1,38 @@
+from blackbox.config import Blackbox as CONFIG
+from blackbox.utils.logger import log
+
+
+class SanitizeReportMixin:
+    """
+    Mixin class helper to mask all sensitive credentials
+
+    Could handle any string with sensitive output
+    """
+
+    def sanitize_output(self, sensitive_output: str) -> str:
+        """ Replace all self.config credentials with in any str *** """
+
+        # first of all let's combine list of secrets from config
+        sensitive_words = []
+        mixed_secrets = [CONFIG.databases.values(), CONFIG.storage.values(),
+                         CONFIG.notifiers.values(), ]
+        for mixed_secret in mixed_secrets:
+            self.extract_secrets(sensitive_words, mixed_secret)
+
+        # now it's time to mask all secrets in our output
+        for sensitive_word in set(sensitive_words):
+            sensitive_output = sensitive_output.replace(
+                sensitive_word, "*" * len(sensitive_word))
+
+        # emphasize output is clear now
+        sanitized_output = sensitive_output
+        log.success("Secrets are masked!")
+        return sanitized_output
+
+    @staticmethod
+    def extract_secrets(sensitive_words: [], mixed_secret) -> str:
```

Comment:
Don't use `[]` as a type annotation. Use `list`.

Reply:
Also, as far as I can tell, this is never used outside this class, so let's not expose it to the handlers. Let's prefix this method with an underscore so that it's clear that it's meant to be a private method.

```suggestion
    def _extract_secrets(sensitive_words: list, mixed_secret) -> str:
```